libkbfs/md_ops.go

```diff
@@ -308,6 +306,27 @@ func (md *MDOpsStandard) processMetadata(ctx context.Context,
 func (md *MDOpsStandard) GetForHandle(ctx context.Context, handle *TlfHandle,
 	mStatus kbfsmd.MergeStatus, lockBeforeGet *keybase1.LockID) (
 	id tlf.ID, rmd ImmutableRootMetadata, err error) {
+	// If we already know the tlf ID, we shouldn't be calling this
+	// function.
```

Comment:
If I understand correctly, this would now only be used for classic TLFs, and will be deprecated after we migrate everything to implicit teams? Can we mention it in its comment in MDOps interface?

Reply:
In the next PR, I actually remove all uses of this function outside of MDOps implementations.  So I can probably make a PR to unexport it and remove it from the interface entirely.

Reply:
(I'll add a TODO in this PR for that.)